.gitignore

```diff
@@ -13,7 +13,6 @@
 server.js
 firebase.js
 
-
  
 # secrets
 **/service-account.json
```

Comment:
please regenerate service-account again. Google reported it was found here: [https://github.com/danesh-23/samwise-positivity/blob/98fc367ac253f7aae3455ca9f3ff93def9379e9d/ServiceAccountKey.json](https://github.com/danesh-23/samwise-positivity/blob/98fc367ac253f7aae3455ca9f3ff93def9379e9d/ServiceAccountKey.json)

Reply:
I have regenerated a ServiceAccount.json file but not included it since we do not need it now.